### PR TITLE
fix(signup): integrate doc upload

### PR DESCRIPTION
**Pull request by MatoPlus**
### Notion ticket link

[Add temp favicon from SDC logo](https://www.notion.so/uwblueprintexecs/Add-temp-favicon-from-SDC-logo-9cbec6fb640342ab903d9645513d77e7)
[Integrate Document Upload for signup flow](https://www.notion.so/uwblueprintexecs/Integrate-Document-Upload-for-signup-flow-beb313f772a2405c96e41102f7b6e426)

https://user-images.githubusercontent.com/44826218/137835621-12eb6fb8-13ea-4860-88ac-9e843ed1eb87.mp4

### Implementation description

- Add page query param for signup pages
- Cleanup signup pages
  - Better placeholders
  - Remove redundant code
- Fix close button bug
- Add favicon from press kit

### Steps to test

1. Sign in and sign up as volunteer
2. Sign in and sign up as parent

### What should reviewers focus on?

- Dev: Style and logic
- Design: Swapped form page for volunteer (for better api consistency)

### Checklist

-   [X] My PR name is descriptive and in imperative tense
-   [X] I have run the linter
-   [X] I have requested a review from the PL, and/or other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: pages/document-upload.tsx [hunk 1]
```diff
@@ -133,7 +133,9 @@ export default function documentUpload({
     const uploadSuccessUI = (): JSX.Element => {
         return (
             <Wrapper session={session}>
-                <CloseButton href={`${redirect as string}`} />
+                <CloseButton
+                    href={redirect ? (redirect as string) : undefined}
```
Comment (gregmaxin) [inline pages/document-upload.tsx:137]: Does `undefined` not result in an error? If there is a case where the close button has an undefined `href` would we instead want to have a default route there vs no behaviour?

Comment (gregmaxin) [reply pages/document-upload.tsx:137]: After reading on I saw 
```
    if (!onClick && !href) {
        onClick = () => router.back();
    }
 ```
 I guess there is a default behaviour if it is undefined :)

**Review by gregmaxin — APPROVED** [review]

**Comment by gregmaxin** [comment]
There is no error message/alert if the file upload fails. Instead, the user is stuck on the page but a submit button that doesn't seem to do anything

**Comment by MatoPlus** [comment]
> There is no error message/alert if the file upload fails. Instead, the user is stuck on the page but a submit button that doesn't seem to do anything

We will refine the doc upload page later!

**Pull request merged**